Userland/Libraries/LibGUI/GlyphMapWidget.cpp

```diff
@@ -178,9 +191,9 @@ void GlyphMapWidget::mouseup_event(GUI::MouseEvent& event)
     if (auto maybe_glyph = glyph_at_position(event.position()); maybe_glyph.has_value()) {
```

Comment:
Since dragging ends if mouse is released outside the map now, we need to constrain the mouseup event to the map rect. Otherwise it can over-select available glyphs. Something like

```suggestion
    auto constrained = event.position().constrained(rect().shrunken(0, frame_thickness() * 2));
    if (auto maybe_glyph = glyph_at_position(constrained); maybe_glyph.has_value()) {
```

Reply:
Fixed.